libr/core/cmd_debug.c

```diff
@@ -2023,6 +2024,34 @@ static void r_core_cmd_bp(RCore *core, const char *input) {
 					   eprintf ("Use: dbc [addr] [command]\n");
 				   }
 				   break;
+		case 'C': // "dbC"
+				   if (input[2] == ' ') {
```

Comment:
if you just need a callback to run commands use R_API int r_core_bind(RCore *core, RCoreBind *bnd);